### PR TITLE
Added registration form with labels

### PR DESCRIPTION
**Pull request by aakirahin**

### CODE REVIEW

File: pubspec.lock [hunk 1]
```diff
@@ -155,7 +169,11 @@ packages:
       name: test_api
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "0.4.8"
+<<<<<<< HEAD
```
Comment (al-anan) [inline pubspec.lock:172]: unresolved conflict, one version needs to be chosen

File: lib/registration.dart/registration_page.dart [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+            FormBuilderTextField(
+              decoration: const InputDecoration(
+                labelText: 'Password',
+              ),
+              name: "Password",
+              validator: (value) {
+                if (value == null || value.isEmpty) {
+                  return "Please enter a password name";
+                } else if (_password.hasMatch(value) == false) {
+                  return "Passwords should have at least one uppercase and lowercase letter, one number and one special character";
```
Comment (al-anan) [inline lib/registration.dart/registration_page.dart:72]: error message too long, does not appear in full when mobile page is in vertical position (looks fine when in horizontal position)

File: lib/screens/landing_screen.dart [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+            padding: const EdgeInsets.all(5.0),
+            child: AppButton(text: 'About us'),
+          ),
+          Padding(
+            padding: const EdgeInsets.all(5.0),
+            child: AppButton(text: 'Login'),
+          ),
+          Padding(
+            padding: const EdgeInsets.all(5.0),
+            child: AppButton(text: 'Create an account'),
```
Comment (al-anan) [inline lib/screens/landing_screen.dart:49]: needs to have a route to Register screen

**Review by al-anan — CHANGES_REQUESTED** [review]

File: pubspec.lock (hunk 1 above)
Comment (aakirahin) [reply pubspec.lock:172]: 0.4.3 is the one we want I think

File: lib/screens/landing_screen.dart (hunk 3 above)
Comment (aakirahin) [reply lib/screens/landing_screen.dart:49]: added

**Review by al-anan — APPROVED** [review]

**Pull request merged**